wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -14,18 +146,17 @@ class AbAlternateContent {
   public function __construct() {
     add_action('the_content', [$this, 'textAlternate']);
   }
+
   function textAlternate($text) {
-    $textToAlter = [
-      'leadpages',
-      'unbounce'
-    ];
-    $alterText = 'Instapage';
+    $textToAlter = get_option('ab_alterer_option_group')['alterFrom'];
+    $alterText = get_option('ab_alterer_option_group')['alterTo']; 
     $alteredText = str_ireplace(
       $textToAlter,
       $alterText,
       $text
     );
+
     return $alteredText;
   }
 }
-$abAlterContent = new AbAlternateContent();
+$abAlternateContentDo = new AbAlternateContent();

```

Comment:
you don't need to store the object. When you just want the constructor to execute, use only 'new ClassName()'